{
  "logs": {
    "2019-02-27T00:00:00-08:00": [
      {
        "date": "2019-02-27T17:26:43.052041-08:00",
        "exc": "",
        "levelname": "INFO",
        "lineno": 51,
        "msg": "File is up.",
        "pathname": "/home/pi/filefairy/tasks/upload/upload.py"
      },
      {
        "date": "2019-02-27T17:28:47.878995-08:00",
        "exc": "",
        "levelname": "INFO",
        "lineno": 84,
        "msg": "Download started.",
        "pathname": "/home/pi/filefairy/tasks/download/download.py"
      },
      {
        "date": "2019-02-27T17:40:18.267824-08:00",
        "exc": "",
        "levelname": "INFO",
        "lineno": 71,
        "msg": "Download finished.",
        "pathname": "/home/pi/filefairy/tasks/download/download.py"
      },
      {
        "date": "2019-02-27T17:44:00.237005-08:00",
        "exc": "",
        "levelname": "INFO",
        "lineno": 166,
        "msg": "Download complete.",
        "pathname": "/home/pi/filefairy/tasks/statsplus/statsplus.py"
      }
    ],
    "2019-03-01T00:00:00-08:00": [
      {
        "date": "2019-03-01T15:56:38.504005-08:00",
        "exc": "Traceback (most recent call last):\n  File \"impl/filefairy/filefairy.py\", line 269, in _try\n    response = item(*args, **kwargs)\n  File \"/home/pi/filefairy/api/registrable/registrable.py\", line 97, in _run\n    return self._run_internal(**kwargs)\n  File \"/home/pi/filefairy/tasks/upload/upload.py\", line 49, in _run_internal\n    date = self._get_date()\n  File \"/home/pi/filefairy/tasks/upload/upload.py\", line 66, in _get_date\n    d = datetime.datetime.strptime(match.title(), '%b %d %Y %I:%M %p')\n  File \"/usr/local/lib/python3.6/_strptime.py\", line 565, in _strptime_datetime\n    tt, fraction = _strptime(data_string, format)\n  File \"/usr/local/lib/python3.6/_strptime.py\", line 362, in _strptime\n    (data_string, format))\nValueError: time data 'March 1 2019 5:55 Pm' does not match format '%b %d %Y %I:%M %p'",
        "levelname": "ERROR",
        "lineno": 272,
        "msg": "Disabled upload.",
        "pathname": "/home/pi/filefairy/impl/filefairy/filefairy.py"
      },
      {
        "date": "2019-03-01T16:25:37.645735-08:00",
        "exc": "Traceback (most recent call last):\n  File \"impl/filefairy/filefairy.py\", line 269, in _try\n    response = item(*args, **kwargs)\n  File \"/home/pi/filefairy/api/registrable/registrable.py\", line 97, in _run\n    return self._run_internal(**kwargs)\n  File \"/home/pi/filefairy/tasks/upload/upload.py\", line 49, in _run_internal\n    date = self._get_date()\n  File \"/home/pi/filefairy/tasks/upload/upload.py\", line 66, in _get_date\n    d = datetime.datetime.strptime(match.title(), '%b %d %Y %I:%M %p')\n  File \"/usr/local/lib/python3.6/_strptime.py\", line 565, in _strptime_datetime\n    tt, fraction = _strptime(data_string, format)\n  File \"/usr/local/lib/python3.6/_strptime.py\", line 362, in _strptime\n    (data_string, format))\nValueError: time data 'March 1 2019 5:55 Pm' does not match format '%b %d %Y %I:%M %p'",
        "levelname": "ERROR",
        "lineno": 272,
        "msg": "Disabled upload.",
        "pathname": "/home/pi/filefairy/impl/filefairy/filefairy.py"
      }
    ]
  }
}
